src/db/commit.ts

```diff
@@ -98,7 +98,10 @@ export class Commit<M extends Meta = Meta> {
     return commits.filter(c => c.isLocal()) as Commit<LocalMeta>[];
   }
 
-  static async baseSnapshot(hash: Hash, dagRead: dag.Read): Promise<Commit> {
+  static async baseSnapshot(
+    hash: Hash,
+    dagRead: dag.Read,
+  ): Promise<Commit<SnapshotMeta>> {
```

Comment:
Thanks, I wonder if we should remove the default value for the type parameter?

Reply:
I put up https://github.com/rocicorp/replicache/pull/715 for this.